mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala

```diff
@@ -270,7 +270,7 @@ object MLUtils {
    */
   @Since("1.0.0")
   @Experimental
-  def kFold[T: ClassTag](rdd: RDD[T], numFolds: Int, seed: Int): Array[(RDD[T], RDD[T])] = {
+  def kFold[T: ClassTag](rdd: RDD[T], numFolds: Int, seed: Long): Array[(RDD[T], RDD[T])] = {
```

Comment:
It seems reasonable. Since the seed parameter of `kfold` can accept `Int` like `kFold(rdd, 2, 1)`, it might not affect user code.

@jkbradley what do you think?